modules/logging-es-deploy-cli.adoc

```diff
@@ -0,0 +1,319 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/cluster-logging-deploying.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-es-deploy-cli_{context}"]
+= Installing {logging-uc} with Elasticsearch using the CLI
+
+Elasticsearch is a memory-intensive application. By default, {product-title} installs three Elasticsearch nodes with memory requests and limits of 16 GB. This initial set of three {product-title} nodes might not have enough memory to run Elasticsearch within your cluster. If you experience memory issues that are related to Elasticsearch, add more Elasticsearch nodes to your cluster rather than increasing the memory on existing nodes.
+
+
+.Prerequisites
+
+* Ensure that you have the necessary persistent storage for Elasticsearch. Note that each Elasticsearch node requires its own storage volume.
++
+[NOTE]
+====
+If you use a local volume for persistent storage, do not use a raw block volume, which is described with `volumeMode: block` in the `LocalVolume` object. Elasticsearch cannot use raw block volumes.
```

Comment:
I'll recommend to set a link to https://docs.openshift.com/container-platform/4.16/scalability_and_performance/optimization/optimizing-storage.html#recommended-configurable-storage-technology_persistent-storage and or to https://docs.openshift.com/container-platform/4.16/observability/logging/log_storage/logging-config-es-store.html#cluster-logging-elasticsearch-storage_logging-config-es-store where it's pretty clear said that also Using NFS is not supported.

Then, it's not only `volumeMode: block` not supported, it's also NFS as described in other sections, but it should be highlighted when "installing" the first time